google/cloud/rest_options.h

```diff
@@ -56,7 +56,7 @@ struct UserIpOption {
  * applicable.
  */
 struct ServerTimeoutOption {
-  using Type = float;
+  using Type = double;
```

Comment:
I just realized what this was.  Why not `using Type = std::chrono::milliseconds` or some other type that is used to denote durations?  We can convert that to the right units when sending it over HTTP.